pkg/document/template.go

```diff
@@ -274,6 +274,84 @@ func getValuesTableTemplates() string {
 	return valuesSectionBuilder.String()
 }
 
+func getValuesTableSectionedTemplates() string {
```

Comment:
I don't like the approach here. I don't think that the addition of sections should necessitate adding new templates. Instead, I propose the following:

- The existing values table templates will change to render the `Sectioned` templates you've implemented here
- If no section is specified for a value then it belongs to a _default_ section.
- If no value specifies a section, then no section header is rendered for the _default_ section, e.g. the values will look exactly the same as they do today.
- If one ore more values specifies a section, the sections are rendered as you have done here, and any values that do not specify a section will be rendered in a section at the end, titled `Other Values` or something generic like that.

Does this all make sense?

Reply:
Makes sense, and also seems to be a better approach here